bitnami/grafana/templates/configmap.yaml

```diff
@@ -3,6 +3,7 @@ kind: ConfigMap
 metadata:
   name: {{ include "grafana.fullname" . }}-envvars
   labels: {{- include "grafana.labels" . | nindent 4 }}
+    app.kubernetes.io/component: {{ include "grafana.name" . }}
```

Comment:
I would change all the lines to add a static value. The component will always be `grafana`, like in the case of the image renderer where the component is `image-renderer`: https://github.com/bitnami/charts/blob/5a09f0f84ac9077502857ba8b82fa0df03d602ad/bitnami/grafana/templates/image-renderer-deployment.yaml#L7

Reply:
Sure, will do

Reply:
I have just addressed 👍